features/step_definitions/purap/requisition.rb

```diff
@@ -1,6 +1,23 @@
-And /^I create a Requisition with required Delivery and Additional Institutional information populated$/ do
+And /^I create a Requisition with required Chart-Organization, Delivery and Additional Institutional information populated$/ do
   visit(MainPage).requisition
   on RequisitionPage do |req_page|
+    # Check Chart/Org associated with the user; if either one does not exist, use corresponding parameter default value and perform lookup
+    chart,org = req_page.chart_org_readonly.split('/')
+    (chart.nil? || chart.empty?) ? (nil) : (chart = chart.strip())
```

Comment:
For this and the next line, is the goal to set chart and org? If so, refactor as
```ruby
chart = (chart.nil? || chart.empty?) ? nil : chart.strip

# or alternatively and probably more clearly:

chart.strip! unless (chart.nil? || chart.empty?)
```